src/Microsoft.ML.Data/DataLoadSave/Database/DatabaseLoaderCursor.cs

```diff
@@ -75,6 +75,10 @@ public DbCommand Command
                     {
                         _command = Connection.CreateCommand();
                         _command.CommandText = _source.CommandText;
+                        if (_source.CommandTimeOut is {})
```

Comment:
Just curious: what will `_source.CommandTimeOut is {}` do? 😮

Reply:
It means  _source.CommandTimeOut is not null  by following the line 74's style. I'm open to any suggestions:)

Reply:
I see! Interesting, didn't know about this way of checking for nullity.
What line 74 style are you referring to?

Also, I wonder, my understanding is that `CommandTimeOut` is of type `int` both in _source and in _command, so is it possible for it to be null? (i.e. does it make sense for us to check for nullity?).

---
In reply to: [450496228](https://github.com/dotnet/machinelearning/pull/5288#discussion_r450496228) [](ancestors = 450496228)

Reply:
It seems that checking with `is {}` an unitialized int field has the same effect as checking an initialized int field, so I'm not sure what's the goal of this check?

[https://dotnetfiddle.net/Qte4sY](https://dotnetfiddle.net/Qte4sY)

---
In reply to: [450519672](https://github.com/dotnet/machinelearning/pull/5288#discussion_r450519672) [](ancestors = 450519672,450496228)

Reply:
Thanks for the example! I just realized that the Int in C# is merely a primitive type so it does not contain nullity properties. I have changed the type of CommandTimeOut in DatabaseSource from int to int?.